samples/sensor/dps310/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+set(DTC_OVERLAY_FILE "${CMAKE_CURRENT_SOURCE_DIR}/dps310.dts")
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
```

Comment:
This should now be `find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})`.  Recent changed merged in upstream Zephyr.